betamax/fixtures/pytest.py

```diff
@@ -49,6 +52,26 @@ def betamax_recorder(request):
 
 
 @pytest.fixture
+def betamax_recorder(request):
+    """Generate a recorder with a session that has Betamax already installed.
```

Comment:
The docstring should also mention that it does not include any pytest parameters for the test function in the name and that that will change by default in 1.0. Also that if they need that behaviour to check out `betamax_parametrized_recorder`.